test/extended/deployments/deployments.go

```diff
@@ -246,6 +247,29 @@ var _ = g.Describe("deploymentconfigs", func() {
 		})
 	})
 
+	g.Describe("when tagging images", func() {
+		g.AfterEach(func() {
+			failureTrap(oc, "tag-images", g.CurrentGinkgoTestDescription().Failed)
+		})
+
+		g.It("should successfully tag the deployed image [Conformance]", func() {
```

Comment:
Unless this was the older test case - the intent was to test the tagging with a test deployment (even if this actually wasn't testing it before)